### PR TITLE
Add test for query confidence levels

### PR DESCRIPTION
**Pull request by e-edling-ericsson**
### Description of the Change

- Added a test to verify that it is possible to get the latest sucessfull
  confidence level for a specific commit.

### Benefits

We can now show that it is possible to get the latest sucessfull
  confidence level for a specific **commit.**
### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Erik Edling erik.edling@ericsson.com

### CODE REVIEW

File: src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java [hunk 1]
```diff
@@ -384,6 +396,41 @@ public void perform__query_and_filter_with_part_of_path() throws Throwable {
         }
     }
 
+    @Then("^perform query to retrieve and filter out confidence level information$")
+    public void perform_query_to_retrieve_and_filter_out_confidence_level_information() throws Throwable {
+    	final String aggrId = "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43";
```
Comment (J-Binieda-ericsson) [inline src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java]: Indentation

File: src/main/java/com/ericsson/ei/mongodbhandler/MongoDBHandler.java [hunk 2]
```diff
@@ -319,6 +319,11 @@ public void createTTLIndex(String dataBaseName, String collectionName, String fi
         return collection;
     }
 
+    public void createCollection(String dataBaseName, String collectionName) {
```
[the remark this answers is not in the document]
Comment (e-edling-ericsson) [reply src/main/java/com/ericsson/ei/mongodbhandler/MongoDBHandler.java]: It's not, i just forgot to remove it, good find :)

File: src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java [hunk 3]
```diff
@@ -84,10 +92,20 @@ public QueryAggregatedObjectsTestSteps() {
         }
     }
 
+    @Before
```
[the remark this answers is not in the document]
Comment (e-edling-ericsson) [reply src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java]: Fixed

File: src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java [hunk 4]
```diff
@@ -21,6 +23,7 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 
+import cucumber.api.java.Before;
```
Comment (J-Binieda-ericsson) [inline src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java]: Do you need to import this? (cucumber.api.java.Before)

Comment (e-edling-ericsson) [reply src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java]: Removed

**Pull request merged**